tests/lib/cloudfront-test.js

```diff
@@ -2,10 +2,12 @@
 /* eslint-disable no-undef */
 /* eslint-disable no-unused-vars */
 var assert = require('../helpers/assert');
+var RSVP = require('rsvp');
+
 
 
 describe('cloudfront', function() {
-  var CloudFront, mockUi, cloudfrontClient, plugin, subject;
+  var CloudFront, validParams, validResponse, cloudfrontClient, plugin, subject;
```

Comment:
I would prefer that we not change the naming of `validOptions` as it's used consistently throughout the tests of other ember-cli-deploy plugins.

```suggestion
  var CloudFront, validParams, validOptions, cloudfrontClient, plugin, subject;
```